doozerlib/cli/__main__.py

```diff
@@ -2262,9 +2262,6 @@ def operator_metadata_latest_build(runtime, nvr, stream, operator_list):
 
 def main():
     try:
-        if 'REQUESTS_CA_BUNDLE' not in os.environ:
-            os.environ['REQUESTS_CA_BUNDLE'] = '/etc/pki/tls/certs/ca-bundle.crt'
```

Comment:
This part of change was included because I was mostly working with this PR and another (python 3 migration and cleanup) at the same time then I involved some changes from there and didn't drop them. I'll drop  this kind of change.

BTW, I remembered I fixed that issue (but forgot to tell everyone) on buildvm by reinstalling `python-certifi` RPM some day.  The RPM version uses system provided CAs rather than vendored CAs from PyPI.